patterns/hidden-written-by.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Title: Written by
+ * Slug: twentytwentyfive/hidden-written-by
+ * Inserter: no
+ *
+ * @package    WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since      Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"style":{"spacing":{"blockGap":"5px","margin":{"bottom":"var:preset|spacing|60"}}},"fontSize":"small","layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group has-small-font-size" style="margin-bottom:var(--wp--preset--spacing--60)">
```

Comment:
Would there be an issue if we set this spacing to `0,2em`? It looks a bit sharper than with `5px`.

Reply:
Here's the difference:

https://github.com/user-attachments/assets/bcd1ba0f-734b-48dd-a8ec-0c22b751a277

Reply:
Done!